packages/code-du-travail-modeles/src/modeles/conventions/1505_commerces_detail_fruits_legumes/indemnite-licenciement.yaml

```
@@ -0,0 +1,50 @@
+contrat salarié . convention collective . commerces detail fruits legumes . indemnité de licenciement:
+  applicable si: indemnité de licenciement
+  valeur: oui
+
+contrat salarié . convention collective . commerces detail fruits legumes . jusqu'à dix ans ou moins:
+  valeur:
+    le minimum de:
+      - indemnité de licenciement . ancienneté conventionnelle en année
+      - 10 an
+
+contrat salarié . convention collective . commerces detail fruits legumes . au dela de dix ans:
+  somme:
+    - indemnité de licenciement . ancienneté conventionnelle en année
+    - (- 10 an)
+  plancher: 0 an
+
+contrat salarié . convention collective . commerces detail fruits legumes . résultat conventionnel . formule moins de 10 ans:
+  applicable si: indemnité de licenciement . ancienneté conventionnelle requise en année <= 10 an
```

En cours de validation avec le métier mais il faudrait utiliser `indemnité de licenciement . ancienneté conventionnelle en année`

---

Il faut utilisé résultat conventionnelle ici 
```suggestion
  applicable si: indemnité de licenciement . ancienneté conventionnelle en année <= 10 an
```

---

+ 1